index.html

```diff
@@ -41,5 +41,163 @@ <h2>I'am a Software Developer</h2>
        </ul>
 
     </section>
+    <section class="recent-work">
+        <h2>My recent works</h2> 
+
+        <hr>
+        
+        <img src="Images/worksection/hero-image.png" alt="image-hero">
+
+        <h3>Multi-Post Stories</h3>
+
+        <p class="text">A daily selection of privately personalized reads; no accounts or
+             sign-ups required. has been the industry's standard dummy 
+             text ever since the 1500s, when an unknown printer took a
+              standard dummy text.</p>
+
+              <ul>
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <a href="#">See project</a>
+
+            </section>
+
+
+    <section class="work-container">
+        <h6>work articles</h6>
+    
+        <article>
```

Comment:
Please use `div` instead of an `article` tag since these projects are not articles 😀 Following is the definition of an article tag from [W3Schools](https://www.w3schools.com/tags/tag_article.asp):

![image](https://user-images.githubusercontent.com/73714615/185617711-17c2d3f1-75f3-4633-b4aa-76e998d1d592.png)